### ISSUE TITLE
3.0 - Checkboxes in Tables Need Labels

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7752]
### Expected behavior
Inputs should have labels.

### Actual behavior
EDD custom tables which include checkboxes do not have labels for those checkboxes. Affected tables include:
- Orders (the main table)
- Customers
- Discounts
- Order Items (individual order screen)
- Order Adjustments (individual order screen)

### Steps to reproduce the behavior
1) running 3.0, visit any screen listed above
2) inspect a checkbox in the table. There is no label.

### Information (if a specific version is affected):
PHP Version: all
EDD Version (or branch): 3.0
WordPress Version: all

Any other relevant information:
On similar tables, WordPress Core outputs a hidden label for similar checkboxes (see the Downloads list table, for example):

```html
<th scope="row" class="check-column">
	<label class="screen-reader-text" for="cb-select-749">Select Sample Bundle</label>
	<input id="cb-select-749" type="checkbox" name="post[]" value="749">
</th>
```

The EDD Orders table:

```html
<th scope="row" class="check-column">
	<input type="checkbox" name="order[]" value="105">
</th>
```

This should be relatively straightforward to fix, because we already have access to the objects, and should be able to update the related functions. For example:
https://github.com/easydigitaldownloads/easy-digital-downloads/blob/5751f4729ef20d33003291107fb98cd7754c3d9c/includes/admin/payments/class-order-items-table.php#L275-L283

can be updated to:

```php
	public function column_cb( $order_item ) {
		return sprintf(
			'<input type="checkbox" name="%1$s[]" id="%1$s-%2$s" value="%2$s" /><label for="%1$s-%2$s" class="screen-reader-text">%3$s</label>',
			/*$1%s*/
			'order_item',
			/*$2%s*/
			$order_item->id,
			/* translators: the product name */
			sprintf( __( 'Select %s', 'easy-digital-downloads' ), $order_item->product_name )
		);
	}
```
etc.

Related: #7737
